src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/DocumentController.kt

```diff
@@ -54,4 +54,49 @@ class DocumentController(context: ActionContext,
 
         return true
     }
+
+    fun refreshDocuments(): String
+    {
+        var url = context.postData<String>("url")
```

Comment:
Why do we have to deal with the vagaries of dropbox here? The url Kim gets from dropbox doesn't get the zip file? Haven't looked at the other tickets, but is the idea that the user needs to provide the url every time, or are we going to persist it for next time's default? Will probably be annoying for user if we don't persist, but maybe for a future release..

Reply:
From experience, Kim tends to give me a link that has "?dl=0" at the end - probably that's the default when you click "get share link". I think rather than relying on the user to get it right every time it makes sense to build in this robustness since it's a check we know how to do.

Agree re persisting as the next improvement to make.